plugins/statistics.koplugin/main.lua

```diff
@@ -376,13 +379,16 @@ function ReaderStatistics:getDatesFromAll(sdays, ptype)
             table.sort(sorted_performance_in_pages)
             for i, n in pairs(sorted_performance_in_pages) do
                 if ptype == "daily_weekday" then
-                    date_text = os.date("%Y-%m-%d (%a)", n)
+                    short_day_of_week = util.translateShortDayOfWeek(os.date("%a", n))
+                    date_text = os.date("%Y-%m-%d ", n) .. "(" .. short_day_of_week .. ")"
```

Comment:
For multiple string concatenation, it's more efficient to use string.format:

```lua
string.format("%s (%s)",
              os.date("%Y-%m-%d ", n),
              shortDayOfWeekTranslation[os.date("%a", n)])
```

Reason: many dynamic languages like LuaJIT, Python, Java, etc use string interning to improve performance and reduce space usage at runtime. As a side effect, all strings are immutable, which means every string concatenation operation creates a new string. So concatenating multiple strings together with `..` will consume more CPU and memory than a simple string.format or table.concat. The later two will only create one new string.